### PR TITLE
331: Hiatus

### PR DESCRIPTION
**Pull request by pimotte**
To allow Arthur to finish his thesis in peace.

休めます。＝ (You) can take a break.

### CODE REVIEW

**Comment by arthurbik** [comment]
This seems to cause a conflict with my right to vote.

**Comment by MrHug** [comment]
Why? You will be a non-player for all intents and purposes, so that should also remove applicability that says "all players have the right to vote"?

File: README.md [hunk 1]
```diff
@@ -232,6 +232,13 @@ This rule takes precedence over every other rule determining the winner.
 **213** A non-player can become a player by submitting a pull request which adds
 their name and Github handle to the Player List.
 
+**331** *I'll be Back!*
+
+A player may at any time create an issue stating they are on hiatus, with the
+label "Hiatus". As long as this issue exists, this player will be consider
```
Comment (MrHug) [inline README.md]: consider=>considered

**Comment by pimotte** [comment]
Exactly why I formulated it this way.

**Comment by arthurbik** [comment]
Ahhh, I forgot that the word player was in there. Sure.

Maybe you should think of what happens when a player becomes a non-player during a trial.

**Comment by pimotte** [comment]
We can unfreeze the gamestate for that trial and the trial remains in limbo until the player returns.

**Comment by MrHug** [comment]
Sounds good to me

**Comment by MrHug** [comment]
:+1:

**Comment by Jason-Statham** [comment]
More votes are needed!

**Comment by arthurbik** [comment]
But if someone breaks a rule, requiring the gamestate to be restored, this is not optimal. Can you exclude trials started before the creation of the hiatus issue?

**Comment by pimotte** [comment]
Sure. I included Judgments too, for good measure.

File: README.md [hunk 2]
```diff
@@ -232,6 +232,15 @@ This rule takes precedence over every other rule determining the winner.
 **213** A non-player can become a player by submitting a pull request which adds
 their name and Github handle to the Player List.
 
+**331** *I'll be Back!*
+
+A player may at any time create an issue stating they are on hiatus, with the
+label "Hiatus". As long as this issue exists, this player will be considered
```
Comment (MrHug) [inline README.md]: this issue is open?

It should exist forever ;)

**Comment by arthurbik** [comment]
:+1:

**Comment by Jason-Statham** [comment]
More votes are needed!

**Comment by MrHug** [comment]
:+1:

**Comment by Jason-Statham** [comment]
More votes are needed!

**Comment by pimotte** [comment]
With 3/4 votes, this is adopted.

**Pull request merged**